src/theme/components/Layout.tsx

```diff
@@ -47,6 +48,26 @@ const Layout = ({ children, title }: Props): JSX.Element => (
       <GlobalStyle />
       <Helmet title={title} />
       {children}
+      <CookieConsent
```

Comment:
⚠️ we are using the default styles of the component so we are not using the color of the theme.
We should at least use the correct red.
That being said, we are managing a transient display here (only displayed once to the user), so we may keep the style as it is now

Reply:
Yes, it's possible to change the color, but I'm not very inspired, and I would just valid the feature.
We can change it in another PR, or you can change it now, if you want ^^

Reply:
Done in d343650 

<img width="1663" alt="image" src="https://user-images.githubusercontent.com/4921914/128526429-11510825-ef3c-4ff2-aaea-e6019cfcd76c.png">

Reply:
nice